src/lib/core/CHIPConfig.h

```diff
@@ -1368,3 +1364,61 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #ifndef CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS
 #define CHIP_CONFIG_MAX_SUBSCRIPTION_RESUMPTION_STORAGE_CONCURRENT_ITERATORS 2
 #endif
+
+/**
+ * @brief Value used when setting or getting the endpoint in a Scene table
+ * entry.  It indicates that the entry is not in use.
+ */
+#ifndef CHIP_CONFIG_SCENES_TABLE_UNUSED_ENDPOINT_ID
```

Comment:
Why does this need to be configurable?  Is 0x00 used because endpoint 0 is never going to have scenes on it or something?  Why would this not just use kInvalidEndpointId?